pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
Yeah, agree. I just don't want to be in position where I feel my suggestions/review comments were misunderstood or feel that I'm pushing back somebody :-)

And agree, we should do this conversation in person and also agree that there are more important things to discuss :-))